test/jest.config.js

```diff
@@ -11,9 +11,6 @@ const customJestConfig = {
   testMatch: ['<rootDir>/**/*int.spec.ts'],
   setupFilesAfterEnv: ['<rootDir>/jest.setup.js'],
 
-  globalSetup: path.resolve(dirname, './helpers/startMemoryDB.ts'),
```

Comment:
The vitest equivalent of this is what breaks debugging. Breakpoints are simply ignored.